go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.58
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/covalent-indexer-go v1.0.6
-	github.com/ElrondNetwork/elastic-indexer-go v1.2.38
+	github.com/ElrondNetwork/elastic-indexer-go v1.2.39-0.20220824110830-f0c615d54cfc
```

Comment:
proper release? Also please reference in the description the PRs involved in the indexer-go repo

Reply:
done.